test/e2e/framework/volume_util.go

```diff
@@ -61,7 +61,6 @@ const (
 	GlusterfsServerImage string = "gcr.io/google_containers/volume-gluster:0.2"
 	CephServerImage      string = "gcr.io/google_containers/volume-ceph:0.1"
 	RbdServerImage       string = "gcr.io/google_containers/volume-rbd:0.1"
```

Comment:
As these images are bit complex and need lot of effort porting them for other archs. Anyways those images are not used in conformance tests.